### PR TITLE
[#175798680] Add call to pay endpoint in FE and test it

### PR DESCRIPTION
**Pull request by GiovanniMancini**
This PR proposes to add a call to `pay` endpoint of PM to execute a payment on press event of `PAY` button in checkout screen.

#### List of Changes

- Added a call to `pay` endpoint of PM in event listener of action `submit` in `checkout` form
- test of `payUsingPOST` method of autogenerated client of PM in node environment in integration with a _dockerized_ version of PM (*not* reproducible in CI/CD)
- test of `payUsingPOST` method of autogenerated client of PM in node environment in integration with a _stub_ of PM (reproducible in CI/CD)
- test of payment workflow in browser environment in integration with a dockerized version of PM

#### Motivation and Context

This change is required to implement the basic payment workflow in a _success scenario_ (e.g. all steps before the payment succeded)

#### How Has This Been Tested?
The changes have been tested by three suites of integration tests 

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts
Comment (pasqualespica) [inline src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts:149]: This test should run before `should call start session, approve terms and add wallet when Continua is pressed on Credit Card input form`, or not ?

File: src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts
Comment (pasqualespica) [inline src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts:168]: When `Paga` is pressed

File: src/inputcard.ts
Comment (pasqualespica) [inline src/inputcard.ts:240]: it can be done?

**Review by pasqualespica — APPROVED** [review]
A lot a stuffs ! Unit & integration test with a complete payment scenario.
Great work 🚀

File: src/inputcard.ts
Comment (GiovanniMancini) [reply src/inputcard.ts:240]: @pasqualespica We have to assess it during the sprint, but it's needed since the cvv is a required input to `pay`

File: src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts
Comment (GiovanniMancini) [reply src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts:168]: OK, I'll fix it

File: src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts
Comment (GiovanniMancini) [reply src/__integrations__/docker/payWorkflow.browser.integration.docker.test.ts:149]: @pasqualespica OK, I'll fix the test order

**Pull request merged**
